### ISSUE TITLE
If have user still default avatar, after click on avatar will be link on his change

### ISSUE THREAD

**Issue by inzerat** [buddycore/surface-theme#63]
Default avatar of user May have Link on his change

**Comment by buddycore** [comment]
Please explain to me more

**Comment by inzerat** [comment]
icon of photoaparat which indicate so its possible change avatar... after click on icon go to change avatar page

![tempprofilepic_2_ 0](https://cloud.githubusercontent.com/assets/21200726/20446630/fbc32d34-adda-11e6-8f1e-6b03b79c9bb0.png)

**Comment by buddycore** [comment]
Yes I can add this to member profiles. Will update a little later today.
